torch/csrc/profiler/kineto_client_interface.cpp

```diff
@@ -91,26 +100,6 @@ struct RegisterLibKinetoClient {
   RegisterLibKinetoClient() {
     static profiler::impl::LibKinetoClient client;
     libkineto::api().registerClient(&client);
-
-    auto kineto_init = []() {
-      libkineto_init(
-          /*cpuOnly=*/!(at::hasCUDA() || at::hasXPU() || at::hasMTIA()),
-          /*logOnError=*/true);
-      libkineto::api().suppressLogMessages();
-    };
-
-    if (std::getenv("KINETO_USE_DAEMON") != nullptr) {
-      int init_delay_s = get_init_delay();
-      if (init_delay_s > 0) {
-        std::thread t([init_delay_s, kineto_init]() {
```

Comment:
If we are removing init_delay_s we should remove KINETO_DAEMON_INIT_DELAY_S from all areas of the code

Reply:
Thanks, this initial commit is a proposal to show another solution for kineto initialization.   As it's reasonable, I will continue improve this pr as comments advised.